nkms/characters.py

```diff
@@ -515,20 +500,19 @@ def from_rest_url(cls, url):
         response = requests.get(url)
         if not response.status_code == 200:
             raise RuntimeError("Got a bad response: {}".format(response))
-        signing_key_bytes, encrypting_key_bytes =\
-                BytestringSplitter(PublicKey)(response.content,
-                                              return_remainder=True)
+        signing_key_bytes, encrypting_key_bytes = \
+            BytestringSplitter(PublicKey)(response.content,
+                                          return_remainder=True)
         stranger_ursula_from_public_keys = cls.from_public_keys(
-                    signing=signing_key_bytes, encrypting=encrypting_key_bytes)
+            signing=signing_key_bytes, encrypting=encrypting_key_bytes)
         return stranger_ursula_from_public_keys
 
     def attach_server(self, ksize=20, alpha=3, id=None,
                       storage=None, *args, **kwargs):
-
         # TODO: Network-wide deterministic ID generation (ie, auction or
```

Comment:
Is this a blocking issue for testnet?

Let's have a conversation about how to achieve this for testnet sooner than later.

Reply:
No it's not.  In fact it's obsolete as far as I can tell at this moment.